third_party/bazel/cuda_static/build_defs.bzl.dummy

```diff
@@ -0,0 +1,2 @@
+def if_has_cublaslt(if_true, if_false = []):
+    return []
```

Comment:
1. it should return `if_false` instead of `[]`.
2. why not reuse `build_defs.bzl.tpl` ?

Reply:
done